integrationTests/flat-config.test.js

```diff
@@ -0,0 +1,14 @@
+const { version } = require('eslint/package.json');
+const semver = require('semver');
+const runJest = require('./runJest');
+
+// ESLint <8 doesn't support the flat config format at all, so this test would
+// always fail.  But we need the skipped test so we don't get a failure for `1
+// snapshot obsolete`.
+if (semver.satisfies(version, '7')) {
```

Comment:
```suggestion
if (semver.satisfies(version, '< 8')) {
```
so it matches the intention.